### ISSUE TITLE
Caching restSearches

### ISSUE THREAD

**Issue by yohonet** [MISP/MISP#1584]
(Feature request)

Scheduled task "cache_exports" is very nice when it comes to processing all events or all attributes from MISP. But if you need a selection of events with specific criterias, you can't cache results of a call to "restSearch". 

It would be interesting to define a list of scheduled restSearches whose results will be cached so that you can run it at a time where your system is idle (over night, for instance) and still be able to get the results anytime within a resonable timeframe.

This would also be nice to be able to set different php parameters for the scheduled tasks (memory_limit and max_execution_timeout) and the MISP UI (which should be more responsive).

**Comment by iglocska** [comment]
Absolutely. I am working on something that should help with this (a rework of the scheduled tasks that allows parameters).
